### ISSUE TITLE
{Linked:Bug11089146} [Adaptive Cards > Designer] - Narrator is announcing incorrect state as expanded for the expand/ collapse buttons when they are in collapsed state.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8701]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 25951.1010)
Browser Version: Edge Dev 117.0.2033.0
Screen Reader: Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Navigate the screen and observe the issue.

**Actual Result:**
Narrator announces incorrect state as expanded for the expand/ collapse buttons when they are in collapsed state.
Narrator is announcing as Expand Items button expanded, button expanded'.

**Expected Result:**
Narrator should announce the correct state as collapsed for the expand/ collapse buttons when they are in collapsed state.
Narrator should announce as ' Line Items button Collapsed, Edit Additional Details button collapsed'. 

**Note:**
Same issue is repro with NVDA screen reader as well.

**User Impact:**
Visually challenged users who rely on screen readers are getting impacted, as Narrator is announcing incorrect state as expanded for the expand/ collapse buttons when they are in collapsed state.

**WCAG Reference:** 
https://www.w3.org/TR/WCAG21/#name-role-value

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/c0cf2bd7-f1c9-4014-bed1-f768b25d0071

![WCAG4 1 2- Narrator is announcing incorrect state as expanded for the expand collapse buttons when they are in collapsed state](https://github.com/microsoft/AdaptiveCards/assets/95913748/67b68516-bf70-4c42-a818-554bb5bdf42c)

[Test JSON.docx](https://github.com/microsoft/AdaptiveCards/files/12833838/Test.JSON.docx)

### Card JSON

```JSON
Attached in Screenshot section.
```

**Comment by vagpt** [comment]
This bug can be originally tracked using below URL.

https://dev.azure.com/microsoftit/OneITVSO/_workitems/edit/11089146

**Comment by anna-dingler** [comment]
Hi @vagpt,

This expand/collapse button is using `Action.ToggleVisibility` which is a deprecated feature. To get the expected behavior, `Action.ShowCard` should be used.

Alternatively, the Narrator is currently announcing the `altText` of the image. The card author could also manually update the expand/collapse text here when the button is clicked (although switching to Action.ShowCard is recommended).

Let me know if this doesn't solve your issue!
Anna
